cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java

```diff
@@ -188,7 +188,7 @@ protected boolean getInvalidConfiguration() {
         private long lastModified;
```

Comment:
An entry in CHANGES_NEXT_RELEASE should be added. For instance:

```
[cygnus-ngsi][NGSINameMappingsInterceptor] Now namemapping checks sevice, subervice and (type of entity and id entity) of EntityMapping (#1535)
```

or another wording if my proposed one is wrong.

that the namemapping checks sevice, subervice and (type of entity and id entity) of EntityMapping.

Reply:
Done